src/styles/custom.scss

```diff
@@ -1,14 +1,20 @@
+@import "~bootstrap/scss/bootstrap";
+
 $body-bg: #F8F9FA;
 $montserrat: 'Montserrat', sans-serif;
 $lora: 'Lora', serif;
 $open-sans: 'Open Sans', sans-serif;
 $main-font-color: #0C2028;
+$body-bg: #F8F9FA;
+$secondary-font-color: #2F4550;
```

Comment:
We no longer need the `$italic-color` and `$secondary-font-color` sass variables here